### ISSUE TITLE
examples/helloworld/tf_example2 seem to quantize on test dataset

### ISSUE THREAD

**Issue by GeLi2000an** [intel/lpot#6]
```
class Dataset(object):
 def __init__(self):'
      (train_images, train_labels), (test_images,
                 test_labels) = keras.datasets.fashion_mnist.load_data()
      self.test_images = test_images.astype(np.float32) / 255.0
      self.labels = test_labels
      pass

  def __getitem__(self, index):
      return self.test_images[index], self.labels[index]

  def __len__(self):
      return len(self.test_images)
```
this code will return test dataset only

**Comment by ftian1** [comment]
yes, in this case we only use test dataset.


`dataset = Dataset()`
`quantizer.metric = common.Metric(MyMetric, 'hello_metric')`
`quantizer.calib_dataloader = common.DataLoader(dataset, batch_size=1)`
`quantizer.eval_dataloader = common.DataLoader(dataset, batch_size=1) `

it means use partial of test images as calibration images(by default it's 100 images) and do evaluatio on full test images.

**Comment by GeLi2000an** [comment]
so far so good!
